chemprop/nn/predictors.py

```diff
@@ -115,27 +115,11 @@ def __init__(
         dropout: float = 0,
         activation: str = "relu",
         criterion: LossFunction | None = None,
-        loc: float | Tensor = 0.0,
-        scale: float | Tensor = 1.0,
     ):
         super().__init__(n_tasks, input_dim, hidden_dim, n_layers, dropout, activation, criterion)
 
-        if isinstance(loc, float):
-            loc = torch.ones(1, self.n_tasks) * loc
-        else:
-            loc = torch.tensor(loc).view(1, -1)
-        self.register_buffer("loc", loc)
-
-        if isinstance(scale, float):
-            scale = torch.ones(1, self.n_tasks) * scale
-        else:
-            scale = torch.tensor(scale).view(1, -1)
-        self.register_buffer("scale", scale)
-
     def forward(self, Z: Tensor) -> Tensor:
-        Y = super().forward(Z)
-
-        return self.scale * Y + self.loc
+        return super().forward(Z)
```

Comment:
this function definition does nothing. That is, simply not defining the function does the same thing as the current definition